elfpy/hyperdrive_interface/hyperdrive_addresses.py

```diff
@@ -11,5 +11,7 @@ class HyperdriveAddresses:
     # pylint: disable=too-few-public-methods
 
     base_token: str = attr.ib()
+    factory: str = attr.ib()
```

Comment:
Can you check that these changes don't break the fetch function here:
https://github.com/delvtech/elf-simulations/blob/main/elfpy/hyperdrive_interface/hyperdrive_interface.py#L28

We've fallen behind on tests in the last week.